src/python/pants/backend/terraform/lint/tfsec/tfsec.py

```diff
@@ -65,17 +77,38 @@ def config_request(self) -> ConfigFilesRequest:
             specified=self.config,
             specified_option_name=f"[{self.options_scope}].config",
             discovery=self.config_discovery,
+            check_existence=[".tfsec/config.json", ".tfsec/config.yml", ".tfsec/config.yaml"],
+        )
+
+    def custom_checks_request(self) -> ConfigFilesRequest:
+        return ConfigFilesRequest(
+            specified=self.custom_check_dir,
+            specified_option_name=f"[{self.options_scope}].custom_check_dir",
+            discovery=self.config_discovery,
             check_existence=[".tfsec/*.json", ".tfsec/*.yml", ".tfsec/*.yaml"],
```

Comment:
Considering this will also include the `config.*` files, do we need to have the two separate? Their only purpose is to find files and include them in the input digest. `tfsec` will then decide what to read for what purpose.

Reply:
The core of the issue is that specifying a value for the setting ("specified") means that discovery doesn't happen. So if someone set this setting to their config file, the custom checks wouldn't be pulled in. Splitting them seemed natural, since they map to separate options on tfsec.